GREEDY.UnitTests/WritingToFileControllerUnitTest.cs

```diff
@@ -3,33 +3,90 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.IO;
+using System.Configuration;
+//using Moq;
+using GREEDY;
+using GREEDY.Interfaces;
+using Telerik.JustMock;
 
 namespace GREEDY.UnitTests
 {
     [TestClass]
     public class WritingToFileControllerUnitTest
     {
+
+        private const string _GoodFileName = "TestReceipt.txt";
+        private const string _BadFileName = "";
+
         [DataTestMethod]
+        [DataRow("1545")]
+        [DataRow("     ")]
         [DataRow("")]
-        [DataRow("   ")]
-        [DataRow("123")]
-        public void WriteToFileTest (string data)
+        [DataRow("123456 cdcd s")]
+
+        public void WriteTextToGoodFile (string data)
```

Comment:
Test methods should contain the word Test in the end

Reply:
not really

Reply:
Well yeah we change our stadard and write differently (nothing to change since we have no Unit Tests). But WriteTextToGoodFile would be better as WriteTextToExistingFile, because it is not know what "good" means at first sight.